templates/overview.html

```diff
@@ -177,7 +177,7 @@ <h4>Use Buffalo at work?</h4>
       <div class="text-center">
         <h2>Want to know more about Buffalo?</h2>
         <p>Get all the support and resources in our documentation.</p>
-        <a class="learn-more-btn" href="https://blog.gobuffalo.io/" target="_blank" rel="noopener noreferrer">Go to the documentation</a>
+        <a class="docs-btn" href="<%= docPath({name: "index", lang: lang}) %>">Go to the documentation</a>
```

Comment:
Can you fix the French version too? https://github.com/gobuffalo/gobuffalo/blob/master/templates/overview.fr.html#L178 Thanks!

Reply:
Done.